package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "src/index.js",
   "sideEffects": false,
+  "browserslist": "> 5%, iOS >= 9, not dead",
```

Comment:
`not dead` ??

Reply:
that's how browserlist specifies it, filters out some ie versions I imagine